source/blog/2011-12-20-doctrine2-2-beta.md

```diff
@@ -36,11 +36,11 @@ the new features. Additionally we try to drive the bug count down in the
 
 Please test this beta with your projects to find any incompatibilities.
 See the
-[UPGRADE\_2\_2](https://github.com/doctrine/doctrine2/blob/master/UPGRADE_TO_2_2)
+[UPGRADE\_2\_2](https://github.com/doctrine/orm/blob/master/UPGRADE_TO_2_2)
```

Comment:
https://github.com/doctrine/orm/blob/master/UPGRADE.md#upgrade-to-22

The 2.2 branch doesn't highlight markdown formats because of a missing file suffix, which is why I use the master-Branch here.